src/misc/interfaces.ts

```diff
@@ -0,0 +1,16 @@
+export interface IAppCategory {
+    title: string;
+    description: string;
+    name: string;
+    value: string;
+}
+export interface IServerInfo1 {
```

Comment:
As mentioned above, please change them into more detailed naming with their actual meaning instead of just adding a `digit` suffix.